Src/FluentAssertions/Common/ConfigurationStoreExceptionInterceptor.cs

```diff
@@ -0,0 +1,36 @@
+﻿#if !NETSTANDARD1_3 && !NETSTANDARD1_6
+
+namespace FluentAssertions.Common
+{
+    internal class ConfigurationStoreExceptionInterceptor : IConfigurationStore
+    {
+        private bool throwsException;
```

Comment:
I would name this `storeThrowsException` or `underlyingStoreUnavailble`. It's about the underlying store, not the current instance.